docs/yubikey_setup.rst

```diff
@@ -1,85 +1,94 @@
-save the log file.
-
-When the configuration has been written, you should see
-green text saying "YubiKey configured" at the top of the
-window.
-
-Set up a user with the OATH-HOTP secret key
--------------------------------------------
-
-Now you will have to set up a new user for the Journalist Interface with
-the secret key from the "Secret Key (20 bytes hex)" field.
-
-manage.py
-~~~~~~~~~
-
-If you have just installed SecureDrop, you will need to add the first
-admin user to the Journalist Interface with ``manage.py``. ``cd`` to the
-``SECUREDROP_ROOT``, which is ``/vagrant/securedrop`` in development and
-``/var/www/securedrop`` in production. Run ``./manage.py add-admin``.
-Fill in the username prompt. Be sure to save the automatically generated
-diceware passphrase in the user's KeePassX database. When it asks "Is this admin
-using a YubiKey [HOTP]? (y/N)", type "y", then enter. At the "Please
-configure your YubiKey and enter the secret:" prompt, enter the Secret
-Key value and hit Enter. Note that the spaces are optional. When you are
-done, you should see a message saying "Admin '(your username)' added".
-
-Admin Interface
-~~~~~~~~~~~~~~~
-
-If you already have an admin user configured, use the "Add user" page in
-the admin interface to add new users. If they want to use YubiKey for
-two-factor, just check the "I'm using a YubiKey [HOTP]" checkbox and
-enter the Secret Key in the "HOTP secret" field.
-
-.. |YubiKey OATH-HOTP Configuration| image:: images/yubikey_oath_hotp_configuration.png
+Using a YubiKey with the Journalist Interface
+=============================================
+
+This guide describes how to set up a YubiKey for two-factor
+authentication on the *Journalist Interface*. This setup is performed
+once per journalist to create a secure method of logging in. The
+process requires some configuration steps using a separate software
+tool.
+
+.. note:: You will do all of these steps from within the Tails
+          operating system.
+
+YubiKey
+-------
+
+A YubiKey is a physical token used for two-factor authentication. They
+are made by a company called Yubico, and are `commercially available
+<https://www.yubico.com/products/yubikey-hardware/fido-u2f-security-key>`__.
+
+Download and Launch the YubiKey Personalization Tool
+----------------------------------------------------
+
+1. Start Tails. At the log in-screen, choose the option to allow an
+   administrator password.
+2. Open a terminal and enter
+
+.. code:: sh
+
+   sudo apt-get install yubikey-personalization-gui
+   
+3. Once you have downloaded and installed the personalization program,
+   open it with administrative privileges by entering
+
+.. code:: sh
+
+   gksu yubikey-personalization-gui
+
+Setting Up Hardware-Based Codes
+-------------------------------
+
+After opening the personalization tool, click the heading
+"OATH-HOTP". This will bring you to a window called "Program in
+OATH-HTOP mode".
```

Comment:
Changed "OATH-HOTP" to **OATH-HOTP**. From the [docs style guide](https://docs.securedrop.org/en/stable/development/documentation_guidelines.html#style-guide):

> When specific elements from a user interface are mentioned by name or by label, **bold** it.

All instances of quotes rather than bold should be made throughout the document, for consistency with the rest of the docs. Again, I realize you didn't introduce these formatting problems, they're vestigial—but let's get the document up to snuff with our present standards!